<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1172</width>
    <height>638</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="1" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="EngineWidget" name="m_engine" native="true">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="widget_2" native="true">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="maximumSize">
         <size>
          <width>600</width>
          <height>600</height>
         </size>
        </property>
        <layout class="QGridLayout" name="gridLayout_4">
         <item row="2" column="0">
          <layout class="QHBoxLayout" name="horizontalLayout_3">
           <item>
            <widget class="QPushButton" name="pushButton_7">
             <property name="text">
              <string>Play</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="pushButton_8">
             <property name="text">
              <string>Pause</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="pushButton_6">
             <property name="text">
              <string>Speed x 2</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="pushButton_5">
             <property name="text">
              <string>Edit</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item row="3" column="0">
          <widget class="QGroupBox" name="groupBox_2">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>100</height>
            </size>
           </property>
           <property name="title">
            <string>Additional</string>
           </property>
           <layout class="QGridLayout" name="gridLayout_3">
            <item row="0" column="0">
             <layout class="QVBoxLayout" name="verticalLayout_3">
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_4">
                <item>
                 <widget class="QLabel" name="label">
                  <property name="text">
                   <string>Scale:</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QSlider" name="horizontalSlider">
                  <property name="sizePolicy">
                   <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                    <horstretch>0</horstretch>
                    <verstretch>0</verstretch>
                   </sizepolicy>
                  </property>
                  <property name="orientation">
                   <enum>Qt::Horizontal</enum>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QCheckBox" name="m_show_grid">
                  <property name="text">
                   <string>Show Grid</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QCheckBox" name="checkBox_2">
                  <property name="text">
                   <string>Show Electro Static Field </string>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_5">
                <item>
                 <widget class="QLabel" name="label_2">
                  <property name="text">
                   <string>Camera at:</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QComboBox" name="comboBox"/>
                </item>
                <item>
                 <widget class="QLabel" name="label_3">
                  <property name="text">
                   <string>Change scene:</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QComboBox" name="comboBox_2"/>
                </item>
               </layout>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
         </item>
         <item row="0" column="0">
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <item>
            <widget class="QGroupBox" name="groupBox">
             <property name="maximumSize">
              <size>
               <width>16777215</width>
               <height>175</height>
              </size>
             </property>
             <property name="title">
              <string>Charges</string>
             </property>
             <layout class="QGridLayout" name="gridLayout_2">
              <item row="0" column="0">
               <layout class="QVBoxLayout" name="verticalLayout_2">
                <item>
                 <widget class="QPushButton" name="pushButton">
                  <property name="text">
                   <string>Add Charge</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QPushButton" name="pushButton_3">
                  <property name="text">
                   <string>Ignore Charges</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QPushButton" name="pushButton_4">
                  <property name="text">
                   <string>Edit Charge</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QPushButton" name="pushButton_2">
                  <property name="text">
                   <string>Remove Charge</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
             </layout>
            </widget>
           </item>
          </layout>
         </item>
         <item row="1" column="0">
          <widget class="QTabWidget" name="tabWidget">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>220</height>
            </size>
           </property>
           <property name="currentIndex">
            <number>0</number>
           </property>
           <widget class="QWidget" name="tab">
            <attribute name="title">
             <string>Position Info</string>
            </attribute>
            <layout class="QGridLayout" name="gridLayout_5">
             <item row="1" column="0">
              <layout class="QHBoxLayout" name="horizontalLayout_6">
               <item>
                <layout class="QVBoxLayout" name="verticalLayout_5">
                 <item>
                  <widget class="QCheckBox" name="m_use_cursor_pos_info">
                   <property name="text">
                    <string>Use Cursor Position</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QGroupBox" name="groupBox_3">
                   <property name="title">
                    <string>Position</string>
                   </property>
                   <layout class="QGridLayout" name="gridLayout_6">
                    <item row="0" column="0">
                     <layout class="QGridLayout" name="gridLayout_7">
                      <item row="0" column="0">
                       <widget class="QLabel" name="m_pos_x_info_label">
                        <property name="text">
                         <string>TextLabel</string>
                        </property>
                       </widget>
                      </item>
                      <item row="1" column="0">
                       <widget class="QLabel" name="m_pos_y_info_label">
                        <property name="text">
                         <string>TextLabel</string>
                        </property>
                       </widget>
                      </item>
                      <item row="1" column="1">
                       <widget class="QLineEdit" name="m_pos_y_info_edit"/>
                      </item>
                      <item row="0" column="1">
                       <widget class="QLineEdit" name="m_pos_x_info_edit"/>
                      </item>
                      <item row="0" column="2">
                       <widget class="QComboBox" name="m_pos_x_info_box"/>
                      </item>
                      <item row="1" column="2">
                       <widget class="QComboBox" name="m_pos_y_info_box"/>
                      </item>
                     </layout>
                    </item>
                   </layout>
                  </widget>
                 </item>
                </layout>
               </item>
               <item>
                <widget class="QGroupBox" name="groupBox_5">
                 <property name="title">
                  <string>Info</string>
                 </property>
                 <property name="checkable">
                  <bool>false</bool>
                 </property>
                 <property name="checked">
                  <bool>false</bool>
                 </property>
                 <layout class="QGridLayout" name="gridLayout_9">
                  <item row="0" column="0">
                   <layout class="QGridLayout" name="gridLayout_8">
                    <item row="2" column="2">
                     <widget class="QComboBox" name="m_tension_y_label_box"/>
                    </item>
                    <item row="2" column="0">
                     <widget class="QLabel" name="m_tension_y_label">
                      <property name="text">
                       <string>TextLabel</string>
                      </property>
                     </widget>
                    </item>
                    <item row="3" column="1">
                     <widget class="QLabel" name="m_potential_label_val">
                      <property name="text">
                       <string>TextLabel</string>
                      </property>
                     </widget>
                    </item>
                    <item row="1" column="0">
                     <widget class="QLabel" name="m_tension_x_label">
                      <property name="text">
                       <string>TextLabel</string>
                      </property>
                     </widget>
                    </item>
                    <item row="2" column="1">
                     <widget class="QLabel" name="m_tension_y_label_val">
                      <property name="text">
                       <string>TextLabel</string>
                      </property>
                     </widget>
                    </item>
                    <item row="1" column="1">
                     <widget class="QLabel" name="m_tension_x_label_val">
                      <property name="text">
                       <string>TextLabel</string>
                      </property>
                     </widget>
                    </item>
                    <item row="3" column="2">
                     <widget class="QComboBox" name="m_potential_label_box"/>
                    </item>
                    <item row="0" column="1">
                     <widget class="QLabel" name="m_tension_val_label_val">
                      <property name="text">
                       <string>TextLabel</string>
                      </property>
                     </widget>
                    </item>
                    <item row="0" column="2">
                     <widget class="QComboBox" name="m_tension_val_label_box"/>
                    </item>
                    <item row="1" column="2">
                     <widget class="QComboBox" name="m_tension_x_label_box"/>
                    </item>
                    <item row="3" column="0">
                     <widget class="QLabel" name="m_potential_label">
                      <property name="text">
                       <string>TextLabel</string>
                      </property>
                     </widget>
                    </item>
                    <item row="0" column="0">
                     <widget class="QLabel" name="m_tension_val_label">
                      <property name="text">
                       <string>TextLabel</string>
                      </property>
                     </widget>
                    </item>
                    <item row="4" column="0">
                     <widget class="QLabel" name="m_energy_val_label">
                      <property name="text">
                       <string>TextLabel</string>
                      </property>
                     </widget>
                    </item>
                    <item row="4" column="1">
                     <widget class="QLabel" name="m_energy_val_label_val">
                      <property name="text">
                       <string>TextLabel</string>
                      </property>
                     </widget>
                    </item>
                    <item row="4" column="2">
                     <widget class="QComboBox" name="m_energy_val_label_box"/>
                    </item>
                   </layout>
                  </item>
                 </layout>
                </widget>
               </item>
              </layout>
             </item>
            </layout>
           </widget>
           <widget class="QWidget" name="tab_2">
            <attribute name="title">
             <string>Work</string>
            </attribute>
            <layout class="QHBoxLayout" name="horizontalLayout_9">
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_8">
               <item>
                <layout class="QHBoxLayout" name="horizontalLayout_7">
                 <item>
                  <layout class="QVBoxLayout" name="verticalLayout_6">
                   <item>
                    <widget class="QCheckBox" name="m_use_cursor_start">
                     <property name="text">
                      <string>Use Cursor Position</string>
                     </property>
                    </widget>
                   </item>
                   <item>
                    <widget class="QGroupBox" name="groupBox_4">
                     <property name="title">
                      <string>Start</string>
                     </property>
                     <layout class="QGridLayout" name="gridLayout_11">
                      <item row="0" column="0">
                       <layout class="QGridLayout" name="gridLayout_12">
                        <item row="0" column="0">
                         <widget class="QLabel" name="m_pos_x_start_label">
                          <property name="text">
                           <string>TextLabel</string>
                          </property>
                         </widget>
                        </item>
                        <item row="1" column="0">
                         <widget class="QLabel" name="m_pos_y_start_label">
                          <property name="text">
                           <string>TextLabel</string>
                          </property>
                         </widget>
                        </item>
                        <item row="1" column="1">
                         <widget class="QLineEdit" name="m_pos_y_start_edit"/>
                        </item>
                        <item row="0" column="1">
                         <widget class="QLineEdit" name="m_pos_x_start_edit"/>
                        </item>
                        <item row="0" column="2">
                         <widget class="QComboBox" name="m_pos_x_start_box"/>
                        </item>
                        <item row="1" column="2">
                         <widget class="QComboBox" name="m_pos_y_start_box"/>
                        </item>
                       </layout>
                      </item>
                     </layout>
                    </widget>
                   </item>
                  </layout>
                 </item>
                 <item>
                  <layout class="QVBoxLayout" name="verticalLayout_7">
                   <item>
                    <widget class="QCheckBox" name="m_use_cursor_dest">
                     <property name="text">
                      <string>Use Cursor Position</string>
                     </property>
                    </widget>
                   </item>
                   <item>
                    <widget class="QGroupBox" name="groupBox_6">
                     <property name="title">
                      <string>Destination</string>
                     </property>
                     <layout class="QGridLayout" name="gridLayout_13">
                      <item row="0" column="0">
                       <layout class="QGridLayout" name="gridLayout_14">
                        <item row="0" column="0">
                         <widget class="QLabel" name="m_pos_x_dest_label">
                          <property name="text">
                           <string>TextLabel</string>
                          </property>
                         </widget>
                        </item>
                        <item row="1" column="0">
                         <widget class="QLabel" name="m_pos_y_dest_label">
                          <property name="text">
                           <string>TextLabel</string>
                          </property>
                         </widget>
                        </item>
                        <item row="1" column="1">
                         <widget class="QLineEdit" name="m_pos_y_dest_edit"/>
                        </item>
                        <item row="0" column="1">
                         <widget class="QLineEdit" name="m_pos_x_dest_edit"/>
                        </item>
                        <item row="0" column="2">
                         <widget class="QComboBox" name="m_pos_x_dest_box"/>
                        </item>
                        <item row="1" column="2">
                         <widget class="QComboBox" name="m_pos_y_dest_box"/>
                        </item>
                       </layout>
                      </item>
                     </layout>
                    </widget>
                   </item>
                  </layout>
                 </item>
                 <item>
                  <layout class="QGridLayout" name="gridLayout_15">
                   <item row="1" column="1">
                    <widget class="QLineEdit" name="m_work_charge_edit"/>
                   </item>
                   <item row="1" column="2">
                    <widget class="QComboBox" name="m_work_charge_box"/>
                   </item>
                   <item row="2" column="2">
                    <widget class="QComboBox" name="m_work_box"/>
                   </item>
                   <item row="2" column="0">
                    <widget class="QLabel" name="m_work_label">
                     <property name="text">
                      <string>TextLabel</string>
                     </property>
                    </widget>
                   </item>
                   <item row="1" column="0">
                    <widget class="QLabel" name="m_work_charge_label">
                     <property name="text">
                      <string>TextLabel</string>
                     </property>
                    </widget>
                   </item>
                   <item row="2" column="1">
                    <widget class="QLabel" name="m_work_label_val">
                     <property name="text">
                      <string>TextLabel</string>
                     </property>
                    </widget>
                   </item>
                   <item row="0" column="1">
                    <spacer name="verticalSpacer">
                     <property name="orientation">
                      <enum>Qt::Vertical</enum>
                     </property>
                     <property name="sizeType">
                      <enum>QSizePolicy::Fixed</enum>
                     </property>
                     <property name="sizeHint" stdset="0">
                      <size>
                       <width>20</width>
                       <height>85</height>
                      </size>
                     </property>
                    </spacer>
                   </item>
                   <item row="3" column="1">
                    <spacer name="verticalSpacer_2">
                     <property name="orientation">
                      <enum>Qt::Vertical</enum>
                     </property>
                     <property name="sizeType">
                      <enum>QSizePolicy::Fixed</enum>
                     </property>
                     <property name="sizeHint" stdset="0">
                      <size>
                       <width>20</width>
                       <height>40</height>
                      </size>
                     </property>
                    </spacer>
                   </item>
                  </layout>
                 </item>
                </layout>
               </item>
              </layout>
             </item>
            </layout>
           </widget>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1172</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpen"/>
    <addaction name="actionSave"/>
    <addaction name="actionSave_as"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuOptions">
    <property name="title">
     <string>Options</string>
    </property>
    <addaction name="actionTheme"/>
    <addaction name="actionLanguage"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionHow_To"/>
   </widget>
   <widget class="QMenu" name="menuCredits">
    <property name="title">
     <string>Credits</string>
    </property>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuOptions"/>
   <addaction name="menuHelp"/>
   <addaction name="menuCredits"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionOpen">
   <property name="text">
    <string>Open</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>Save</string>
   </property>
  </action>
  <action name="actionSave_as">
   <property name="text">
    <string>Save as</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="actionTheme">
   <property name="text">
    <string>Theme</string>
   </property>
  </action>
  <action name="actionLanguage">
   <property name="text">
    <string>Language</string>
   </property>
  </action>
  <action name="actionHow_To">
   <property name="text">
    <string>How To</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>EngineWidget</class>
   <extends>QWidget</extends>
   <header>Interface/enginewidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>m_show_grid</sender>
   <signal>stateChanged(int)</signal>
   <receiver>MainWindow</receiver>
   <slot>showGrid(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>903</x>
     <y>531</y>
    </hint>
    <hint type="destinationlabel">
     <x>984</x>
     <y>605</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionExit</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>585</x>
     <y>318</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>showGrid(int)</slot>
 </slots>
</ui>
